accelerator/server.cc

```diff
@@ -110,6 +110,20 @@ int main(int, char const**) {
         res << json_result;
       });
 
+  /**
+   * @method {post} /transaction send transfer
+   *
+   * @return {String} transaction object
+   */
+  mux.handle("/transaction")
+      .post([&](served::response& res, const served::request& req) {
+        char* json_result;
+
+        api_send_transfer(&service, req.body().c_str(), &json_result);
```

Comment:
Checks for request header like if its `content-type` is `application/json` are needed since this is a POST method.

Reply:
`req.header("content-type")` is what you need for above example.